src/utils/bot.rs

```diff
@@ -13,3 +13,22 @@ pub fn load_configuration() -> Configuration {
 pub async fn get_data_lock(ctx: &serenity::Context) -> Arc<RwLock<Data>> {
     ctx.data.read().await.get::<Data>().unwrap().clone()
 }
+
+pub async fn get_member(
```

Comment:
Is there a better place for this function?

Reply:
Would'nt [std](https://doc.rust-lang.org/std/index.html)::[result](https://doc.rust-lang.org/std/result/#) be better? You'd then use 
```rs
std::Result<serenity::Member, serenity::SerenityError::Http>
```
as the return value instead of wrapping it in an `Option`.